README.md

```diff
@@ -66,6 +66,31 @@ If you'd like to publish the image to Docker Hub:
 ./docker-run.sh --pushimage
 ```
 
+## Environment variables
+
+When running ontology-publisher it is possible to set environment variables appropriate for the processed ontology as follows:
+
+```bash
+env <VARIABLE_NAME>=<VARIABLE_VALUE> [<VARIABLE_NAME1>=<VARIABLE_VALUE1>...] ./docker-run.sh [OPTION...]
+```
+
+for example:
+
+```bash
+env ONTPUB_FAMILY="onto" ONTPUB_INPUT_REPOS="onto core" ./docker-run.sh --shell --dev
+```
+
+List of allowed `<VARIABLE_NAME>` (and defaults for `<VARIABLE_VALUE>`):
+
+- `ONTPUB_FAMILY` :- ontology name (base for many other values), e.g. `onto` (default: `fibo`); this variable is also possible to set during build time (see `--rebuildimage` option)
+- `ONTPUB_SPEC_HOST` :- the basis of the ontology IRI=`https://${ONTPUB_SPEC_HOST}/${ONTPUB_FAMILY}/ontology/`, e.g. `onto.example.org` (default: `spec.edmcouncil.org`); this variable is also possible to set during build time (see `--rebuildimage` option)
+- `ONTPUB_INPUT_REPOS` :- list of directories (located in the user's home directory) containing input ontologies that will be mounted in the _/input_ directory inside container, e.g. `onto core` (default: `${ONTPUB_FAMILY} LCC`)
+- `DEV_SPEC` :- the name of the file (inside the directory named `${ONTPUB FAMILY}`, in the `${ONTPUB_INPUT_REPOS}` list) containing the "Development" ontology, e.g. `Devel.rdf` (default: `About${ONTPUB_FAMILY}Dev.rdf`)
+- `PROD_SPEC` :- the name of the file (inside the directory named `${ONTPUB FAMILY}`, in the `${ONTPUB_INPUT_REPOS}` list) containing the "Production" ontology, e.g. `Prod.rdf` (default: `About${ONTPUB_FAMILY}Prod.rdf`)
```

Comment:
Since we never have processed, yet, a file name 'Prod.rdf', this name might be confusing as an example. I suggest to rename it 'AboutFiboProc.rdf'. This remark applies to all occurrences of Prod.rdf'.